app/env.js

```diff
@@ -30,6 +30,8 @@ const vars = envalid.cleanEnv(
     LOG_LEVEL: envalid.str({ default: 'info', devDefault: 'debug' }),
     PORT: envalid.port({ default: 80, devDefault: 3000 }),
     NODE_HOST: envalid.host({ devDefault: 'localhost' }),
+    IPFS_API_HOST: envalid.host({ default: 'localhost' }),
```

Comment:
should maybe be `devDefault` since its `localhost`?

Reply:
updated, to be fair we do not use this just yet so changed to a default as did not see a reason to separate, however changed it back.